File: 2.0/python/calls/getCallRecordings.py
from callfire.client import CallfireClient

client = CallfireClient('api-login', 'api-password')
response = client.calls.getCallRecordings(
    id=379506003,
    # return only specific fields
    fields='items(callId,campaignId,lengthInBytes,mp3Url)'
).result()

# see sample JSON response for this API
# on 'curl' samples tab
print(response)

File: 2.0/php/calls/getCallRecordings.php
<?php

class ApiClientSample {

    public static function main() {
        $client = \CallFire\Api\DocumentedClient::createClient("login", "password");
        $request = $client->getCallRecordings();
        $request->getOperationConfig()->setPathParameters(array("id" => 379506003));
        $request->getOperationConfig()->setQueryParameters(array("fields" => "items(callId,campaignId,lengthInBytes,mp3Url)"));
        $result = $client->request($request);
        $json = json_decode($result->getBody());
    }
}

ApiClientSample::main();

